tests/queries/test_query.py

```diff
@@ -165,6 +165,18 @@ def test_equality(self):
             Author.objects.filter(Q(item__name='foo')).query,
         )
 
+    def test_equality_with_negated_nullable(self):
```

Comment:
Please move this test to `tests.queries.tests.ExcludeTests`.

Reply:
@felixxm okay! Thanks for review.